static/app/components/forms/formIndicators.tsx

```diff
@@ -155,7 +157,7 @@ const FieldName = styled('span')`
   font-weight: ${p => p.theme.fontWeight.bold};
   margin: 0 ${space(0.5)};
 `;
-const MessageContainer = styled('div')`
-  display: flex;
-  align-items: center;
-`;
+
+function MessageContainer(props: FlexProps) {
+  return <Flex align="center" {...props} />;
+}
```

Comment:
This could probably be inlined?

Reply:
yeah but they are all in the same component, so we can fix it by defining:

```
const rootArgs = {
  root: <Flex align="center" />,
};
```

and then spread it where it’s used:

```
const tctArgsSuccess = {
  ...rootArgs,
```